sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -40,6 +77,105 @@ export class StudentAESTController extends BaseController {
       metadata: studentDocument.metadata,
       groupName: studentDocument.groupName,
       updatedAt: studentDocument.updatedAt,
+      fileOrigin: studentDocument.fileOrigin,
     }));
   }
+
+  /**
+   * Gets a student file and writes it to the HTTP response.
+   * @param uniqueFileName unique file name (name+guid).
+   * @param response file content.
+   */
+  @Get("files/:uniqueFileName")
+  @ApiNotFoundResponse({ description: "Requested file was not found." })
+  async getUploadedFile(
+    @Param("uniqueFileName") uniqueFileName: string,
+    @Res() response: Response,
+  ): Promise<void> {
+    await this.studentControllerService.writeFileToResponse(
+      response,
+      uniqueFileName,
+    );
+  }
+
```

Comment:
Are we expecting something to be returned here as the method names suggests ?

Reply:
The method writes a file to the HTTP response, so the method itself does not return an object but the endpoints return the file. Does it make sense?

Reply:
@henryoforeh-aot even though the method is not returning anything, it actually changes the response parameter by writing the value into it. So a get call is used.

Reply:
Makes sense.